### ISSUE TITLE
calibrate_chest failed during installation

### ISSUE THREAD

**Issue by ferdianjovan** [strands-project/scitos_common#37]
I just did a fresh install of the strands_catkin_ws. 

There is a problem with scitos_common/calibrate_chest due to this part
**/opt/strands/strands_catkin_ws/src/scitos_common/calibrate_chest/src/calibrate_chest.cpp:33:51: error: no matching function for call to ‘pcl::visualization::PCLVisualizer::addCoordinateSystem(double, const char [10], int)’**

```
/opt/strands/strands_catkin_ws/src/scitos_common/calibrate_chest/src/calibrate_chest.cpp: In function ‘void plot_best_plane(const pcl::PointCloud<pcl::PointXYZ>&, Eigen::Vector4f, double)’:
**/opt/strands/strands_catkin_ws/src/scitos_common/calibrate_chest/src/calibrate_chest.cpp:33:51: error: no matching function for call to ‘pcl::visualization::PCLVisualizer::addCoordinateSystem(double, const char [10], int)’**
/opt/strands/strands_catkin_ws/src/scitos_common/calibrate_chest/src/calibrate_chest.cpp:33:51: note: candidates are:
/usr/include/pcl-1.7/pcl/visualization/pcl_visualizer.h:281:9: note: void pcl::visualization::PCLVisualizer::addCoordinateSystem(double, int)
/usr/include/pcl-1.7/pcl/visualization/pcl_visualizer.h:281:9: note:   candidate expects 2 arguments, 3 provided
/usr/include/pcl-1.7/pcl/visualization/pcl_visualizer.h:291:9: note: void pcl::visualization::PCLVisualizer::addCoordinateSystem(double, float, float, float, int)
/usr/include/pcl-1.7/pcl/visualization/pcl_visualizer.h:291:9: note:   candidate expects 5 arguments, 3 provided
/usr/include/pcl-1.7/pcl/visualization/pcl_visualizer.h:325:9: note: void pcl::visualization::PCLVisualizer::addCoordinateSystem(double, const Affine3f&, int)
/usr/include/pcl-1.7/pcl/visualization/pcl_visualizer.h:325:9: note:   no known conversion for argument 2 from ‘const char [10]’ to ‘const Affine3f& {aka const Eigen::Transform<float, 3, 2>&}’
make[2]: *** [scitos_common/calibrate_chest/CMakeFiles/calibrate_chest.dir/src/calibrate_chest.cpp.o] Error 1
make[1]: *** [scitos_common/calibrate_chest/CMakeFiles/calibrate_chest.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by kunzel** [comment]
I can confirm that I had the same problem on my laptop after an update. As a temporary workaround I placed a CATKIN_IGONORE file in the package..

**Comment by hawesie** [comment]
This is my fault. Go to the line in question and remove the second and third arguments from the methods call and everything will be ok. Please also commit this back. Or just revert the relevant commit of mine to this repo.

**Comment by hawesie** [comment]
In fact, I'll do it now.

**Comment by hawesie** [comment]
Fixed in 92f5d086da2eeafb20062be64ceb37a384930d51
